packages/tsutils/src/validation/ajv/getAjv.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { constructRecordExistsWithId } from '@tupaia/utils';
+import Ajv from 'ajv';
+import addFormats from 'ajv-formats';
+
+import { IdFormat } from './customFormats';
+
+type DatabaseModels = { entity: any; question: any; optionSet: any };
```

Comment:
I guess we are fine to leave it as any? As DatabaseModelType only exists in database package.

Reply:
Yeah sounds good, can deal with it later 👍